CONTRIBUTORS.md

```diff
@@ -413,6 +434,7 @@ GitHub contributors:
 - moncef102
 - montes
 - mplh
+- Mreker
```

Comment:
Please change to: Mathias Reker

Reply:
This information comes from your commit messages @MathiasReker. If you want to be credited differently, consider changing your git configuration.

Reply:
I fixed my git config :)

Reply:
indeed the lidt is generated automatically and then injected manually in the file, I am gonna fix the duplicate for the next build